tools/BUILD

```diff
@@ -0,0 +1,24 @@
+package(
+    default_visibility = ["//visibility:private"],
+)
+
+PIP_DEPS = [
+    "//gematria/llvm/python:canonicalizer.so",
+    "//gematria/llvm/python:llvm_architecture_support.so",
+    "//gematria/basic_block/python:basic_block_protos.so",
+    "//gematria/basic_block/python:basic_block.so",
+    "//gematria/datasets/python:bhive_importer.so",
+    "//gematria/granite/python:graph_builder.so",
+    "//gematria/model/python:oov_token_behavior.so",
+    "//gematria/proto:basic_block_py_pb2",
+    "//gematria/proto:canonicalized_instruction_py_pb2",
+    "//gematria/proto:throughput_py_pb2",
+    "//gematria/utils/python:pybind11_abseil_status"
+]
+
+sh_binary(
```

Comment:
This should be a `gematria_py_binary`, no? Normally, `sh_binary` should be reserved for bash scripts.